pkg/privilege/privileges/tidb_auth_token_test.go

```diff
@@ -417,3 +417,31 @@ func TestJWKSImpl(t *testing.T) {
 	_, err = jwksImpl.checkSigWithRetry(signedTokenString, 0)
 	require.Error(t, err)
 }
+
+func (i *immutable) User() []UserRecord {
```

Comment:
You can rename `i` to `p` to avoid the warning `Receiver names are different`

Reply:
done